include/torch-mlir/Conversion/Passes.td

```diff
@@ -114,4 +114,15 @@ def ConvertTorchToTosa : Pass<"convert-torch-to-tosa", "FuncOp"> {
   let constructor = "mlir::torch::createConvertTorchToTosaPass()";
 }
 
+def ConvertLowerAlgorithmicOps : Pass<"convert-lower-algorithmic-ops", "FuncOp"> {
```

Comment:
Same here that `convert` need to be removed from the pass name. `lower` is a kind of `convert` already.